15/umbraco-cms/extending/creating-custom-seed-key-provider.md

```diff
@@ -0,0 +1,115 @@
+﻿---
+description: A guide to creating a custom seed key provider for Umbraco
+---
+
+# Creating a Custom Seed Key Provider
+
+From version 15 and onwards Umbraco uses a lazy loaded cache, this means content is loaded into the cache on an as-needed basis.
+However, you may some specific content to always be in the cache, to achieve this you can implement your own custom seed key providers.
+
+There is two types of seed key providers: `IDocumentSeedKeyProvider` for documents and `IMediaSeedKeyProvider` for media,
+these interfaces are identical so only `IDocumentSeedKeyProvider` is demonstrated here.
```

Comment:
I suggest we remove the version number. I mean, is it relevant to this article? 🤔 You'll already be viewing the docs for version 15 😅 

```suggestion
Umbraco uses a lazy loaded cache, which means that content is loaded into the cache on an as-needed basis. However, you may need specific content to always be in the cache. To achieve this you can implement your own custom seed key providers.

There are two types of seed key providers: `IDocumentSeedKeyProvider` for documents and `IMediaSeedKeyProvider` for media. As these interfaces are identical only `IDocumentSeedKeyProvider` is demonstrated in this article.
```

Reply:
No that makes a lot of sense 😄